smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/autocompound.rs

```diff
@@ -322,27 +315,24 @@ fn test_autocompound_with_rewards_swap_non_vault_funds() {
 
     let after_swap_rewards_balance =
         get_balance_amount(&app, contract_address.to_string(), DENOM_REWARD.to_string());
-    assert_eq!(0u128, after_swap_rewards_balance,);
     assert_eq!(0u128, after_swap_rewards_balance);
 
     let after_swap_base_balance =
         get_balance_amount(&app, contract_address.to_string(), DENOM_BASE.to_string());
     // Assert vault position tokens balances increased accordingly to the swapped funds from DENOM_REWARD to DENOM_BASE and DENOM_QUOTE
-    assert_approx_eq!(
+    assert_eq!(
         after_deposit_base_balance
-            .checked_add(rewards_swap_amount.into())
+            .checked_add(swap_token_in_amount.into())
```

Comment:
that looks now totally wrong. After a swap I expect `before_swap_balance - swap_token_in_amount = after_swap_balance` and `before_swap_balance + swap_out_amount = after_swap_balance`

Reply:
Yes you right, the math was right as here, working with the assump[tion of spot price = 1.0 we could do that. But the nomenclature was clearly wrong. Fixed!